src/js/stores/UserSettingsStore.ts

```diff
@@ -0,0 +1,61 @@
+import EventEmitter from "events";
+
+import LocalStorageUtil from "../utils/LocalStorageUtil";
+
+const LOCAL_STORAGE_KEY: string = "dcosUserSettings";
+const JSON_EDITOR_SETTING: string = "JSONEditor";
+const isObject = (data: unknown): data is Record<string, unknown> =>
+  typeof data === "object" && data != null;
+
+function getLocalStorageObject() {
+  const localStorageObject = LocalStorageUtil.get(LOCAL_STORAGE_KEY);
+  if (localStorageObject != null) {
```

Comment:
why do we need to introduce this check here?

Reply:
null would occur in situations where a given key is not set in localStorage. The return value of the `localStorage.getItem` is `string | null`. So without the check I was given a TS problem.